pages/integrations/sso/google_workspace_saml.md.erb

```diff
@@ -55,27 +57,29 @@ Click the 'Edit Settings' button, choose the Manual data option, and enter the I
     <tr>
         <td>X.509 certificate</td>
         <td>
-            The public key certificate generated for you by Google Workspace that you downloaded during the previous step. You'll need the whole file, not just a link to the file.
+            The public key certificate generated for you by Google Workspace that you downloaded during the previous step. You need the whole file, not just a link to the file.
         </td>
     </tr>
 </table>
 
-Save your new settings, and you'll be returned to your provider page.
+Save your new settings, and Buildkite returns you to your provider page.
```

Comment:
Not super keen on 

> Buildkite returns you to your provider page.

How about something something confirm that provider page opens? something something returned to your provider page? it's all a bit meh

Reply:
How about:

Save your settings. Buildkite returns you to your provider page.

Or

Save your settings. Your provider page opens.

?

Reply:
```suggestion
Save your settings. Your provider page opens.
```